openedx/features/smart_referral/admin.py

```diff
@@ -11,8 +11,8 @@ class SmartReferralAdmin(admin.ModelAdmin):
     Django admin customizations for SmartReferral model.
     """
 
-    list_display = ('user', 'contact_email', 'is_contact_reg_completed')
-    search_fields = ('user__username', 'contact_email', 'is_contact_reg_completed')
+    list_display = ('user', 'contact_email', 'is_referral_step_complete')
+    search_fields = ('user__username', 'contact_email', 'is_referral_step_complete')
```

Comment:
This flag now behaves differently, we should not show it in the admin. It can lead to false information.

Scenario: A contact gets a referral invite. He signup on the platform. The admin page will show `is_referral_step_complete` as false (until the command is run, once).

Reply:
@zeshanarifios Please review this comment.